website/docs/sql-function-glossary/sql-aggregate-functions/sql-max.md

```diff
@@ -0,0 +1,58 @@
+---
+id: sql-max
+title: SQL MAX
+description: The SQL MAX aggregate function allows you to compute the maximum value from a column.
+slug: /sql-function-glossary/sql-max
+---
+
+<head>
+    <title>Working with the SQL MAX function</title>
+</head>
+
+The SQL MAX aggregate function allows you to compute the maximum value from a column. This kind of measure is useful for understanding the distribution of column values, determining the most recent timestamps of key events, and creating booleans from CASE WHEN statements.
```

Comment:
I understand what you're getting at w/ creating bools from CASE statements, but what's the ultimate purpose of that?  Would you call that normalization or flattening or tokenization or something?

Reply:
Normalization sounds right to me, curious what you think

Reply:
Mmm I don't know if normalization is _quite_ what I associate with this; I think flattening is probably most appropriate

Reply:
Let me know if I added this into the content properly!